modules/mobile/app/controllers/mobile/v0/users_controller.rb

```diff
@@ -30,7 +30,9 @@ def options
       end
 
       def pre_cache_resources
-        Mobile::V0::PreCacheAppointmentsJob.perform_async(@current_user.uuid)
+        if Flipper.enabled?(:mobile_precache_appointments)
```

Comment:
IMO the flag needs to be in the job itself to ensure that there's not a way around the flag, but I also added it in the controllers because I don't want to queue up a bunch of jobs for no reason and add more strain to sidekiq.